pretix_mollie/signals.py

```diff
@@ -101,3 +100,14 @@ def refresh_mollie_tokens(sender, **kwargs):
             if rt not in seen:
                 refresh_mollie_token(es.object, True)
                 seen.add(rt)
+
+
+@receiver(order_modified, dispatch_uid='mollie_order_modified')
```

Comment:
pretix.base.services.orders.extend_order does not seem to trigger this signal

Reply:
Hm, that kinda makes this whole thing obsolete...

I guess we could either introduce a new `expiration_changed` or emit the `order_changed`-signal from the service. Any preference?